kubernetes/charts/weblogic-operator/templates/_operator-dep.tpl

```diff
@@ -64,6 +64,8 @@ spec:
           value: "false"
         - name: "JAVA_LOGGING_LEVEL"
           value: {{ .javaLoggingLevel | quote }}
+        - name: "CLOUD_PLATFORM"
+          value: {{ .cloudPlatform | quote }}
```

Comment:
There technically might not be a cloud involved. How about `.kubernetesPlatform` and `KUBERNETES_PLATFORM` throughout?

Reply:
Also, this setting should be documented in `userguide/managing-operators/using-helm.md`, `security/openshift.md`, and` `userguide/platforms/environments.md#openshift`.